build-support/bin/_release_helper.py

```diff
@@ -816,7 +816,14 @@ def build_pex(fetch: bool) -> None:
         dest = stable_dest
     green(f"Built {dest}")
 
-    subprocess.run([sys.executable, str(dest), "--no-pantsd", "--version"], env=env, check=True)
+    with TemporaryDirectory() as tmpdir:
```

Comment:
I've lost the thread here I think. Are these new gymnastics still needed after Stu's fix?

Reply:
Yeah, they are. We were running in `dist/` and it it was picking up config from `../pants.toml` I think - for whatever reason, running from `dist/` would not cause it to fail. Now we test in a completely hermetic tmpdir.

The other gymnastics with `chmod` are to make sure that your shebang improvement does not regress.